src/app/search/search.html

```diff
@@ -0,0 +1,26 @@
+<div class="gh-search well">
+    <form role="form">
+        <div class="form-group has-feedback">
+            <label class="control-label">Search: </label>
+            <input type="text" class="form-control" placeholder="Search"
+                   ng-model="searchCtrl.query"
+                   ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }"
+                   ng-change="searchCtrl.searchRepos()"/>
+            <i class="form-control-feedback fa fa-search"></i>
+        </div>
+    </form>
+
+    <ol>
+        <li ng-repeat="repo in searchCtrl.repos">
```

Comment:
What reason behind this?
It doesn't make sense to implement it in this case, as for me.
In most cases ng-repeat will be enough, you can use ```track by``` to improve perfomamce